### PR TITLE
Added navigational menu

### PR DESCRIPTION
**Pull request by lorienhall**
I added a sticky navigational menu at the top with links to other spots in the document.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -2,6 +2,23 @@
 <html lang="en">
   <head>
   	<meta charset="utf-8">
+
```
Comment (Mbd06b) [inline docs/index.html:5]: Wrap this list element in a <nav> tag.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav  make sure to give it a title/role/aria-label attribute describing the landmark element.

Comment (Mbd06b) [reply docs/index.html:5]: This is also an invalid location for this list/nav element. The only elements that should be located inside the <head> tag are <title>,<link><meta><script> or <style> tags or similar tags that describe the html. 
Semantic/Landmark elements all need to be contained within the <body></body> tag.  

Solution: Move right below opening <body> tag.

**Review by Mbd06b — CHANGES_REQUESTED** [review]
Fix comments.  Run this HTML source page through W3 validator fix all errors, warnings are ok but feel free to fix them as well.  https://validator.w3.org/

File: docs/index.html (hunk 1 above)
Comment (lorienhall) [reply docs/index.html:5]: I believe I have this. Please let me know if its still off.

File: docs/index.html [hunk 2]
```diff
@@ -38,6 +22,23 @@
...
+				<li><a href="#task2"></a></li>
+				<li><a href="#task2_1"></a></li>
+				<li><a href="#task3"></a></li>
+			</ul>  
+		</li>
+    		<li><a href="#conceptsFromSetup">Concepts from Setup</a></li>      
+		<li><a href="#techStackRef">Tech Stack Reference</a></li>    
+		<li><a href="#projectStructure">Project Structure</a></li>  
+    	</ul>
+  </nav>
```
Comment (Mbd06b) [inline docs/index.html:41]: Still need to move inside the `<body></body>` tag.

Comment (Mbd06b) [reply docs/index.html:41]: and move the .sticky class to the new `<nav>` tag

**Review by Mbd06b — CHANGES_REQUESTED** [review]
Move the `<nav></nav>` into the `<body>` tag, and move .sticky class from the ul to the nav.

**Comment by lorienhall** [comment]
I've fixed all the errors W3 gives me, except for one. On line 210 it says "Element form:form not allowed as child of element pre in this context. (Suppressing further errors from this subtree.)". I'm not sure why, except that the syntax is off, although I have no Idea how to fix it.

**Review by Mbd06b — APPROVED** [review]
Looks good.

**Pull request merged**